artifactory/default.toml

```diff
@@ -0,0 +1,25 @@
+#                 _   _  __           _                    
```

Comment:
I know its just comments, but can we remove the large text? Seems unnecessary.

Reply:
Done...large text has been removed.